packages/shared/components/atoms/tiles/AliasActivityTile.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="typescript">
+    import { localize } from '@core/i18n'
+    import {
+        InclusionState,
+        selectedAccountAssets,
+        getAssetFromPersistedAssets,
+        IPersistedAsset,
+        IAliasActivityData,
+    } from '@core/wallet'
+    import { truncateString } from '@lib/helpers'
+    import { openPopup } from '@lib/popup'
+    import { ClickableTile, Text, AssetIcon } from 'shared/components'
+    import { FontWeight } from 'shared/components/Text.svelte'
+
+    export let activityId: string
+    export let inclusionState: InclusionState
+    export let data: IAliasActivityData
+
+    let asset: IPersistedAsset
+
+    $: $selectedAccountAssets, (asset = getAssetFromPersistedAssets(data.assetId))
+
+    function handleTransactionClick(): void {
+        openPopup({
+            type: 'activityDetails',
+            props: { activityId },
+        })
+    }
+</script>
+
+<ClickableTile
+    onClick={handleTransactionClick}
+    classes={inclusionState === InclusionState.Confirmed ? '' : 'opacity-50'}
+>
+    <div class="w-full flex flex-col space-y-4">
+        <div class="flex flex-row items-center text-left space-x-4">
+            <AssetIcon {asset} showVerifiedBadgeOnly />
+            <div class="flex flex-col w-full space-y-0.5">
+                <div class="flex flex-row">
+                    <Text
+                        fontWeight={FontWeight.semibold}
+                        lineHeight="140"
+                        classes="overflow-hidden overflow-ellipsis multiwrap-line2"
+                    >
+                        {localize(
+                            inclusionState === InclusionState.Confirmed
+                                ? 'general.aliasCreated'
+                                : 'general.creatingAlias'
+                        )}
+                    </Text>
+                </div>
+                <div class="flex flex-row">
+                    <Text fontWeight={FontWeight.normal} lineHeight="140" color="gray-600">
+                        {truncateString(data.aliasId, 20, 0)}
```

Comment:
Any feedback on what we wanna display here? alias id is somehow `0x0000000000000000000000000`

Reply:
We have the output id of the alias and the alias address
I think the 0 is just the internal id of the number of alias in that account

Reply:
Now its bech32 we should display at least the first 5 characters so we can distinguish the address type and then the check sum